snap/snapcraft.yaml

```
@@ -334,6 +332,11 @@ apps:
       - network-bind
 
 parts:
+  version:
+    plugin: nil
+    source: .
```

as I mentioned over telegram, let's use this instead:

```suggestion
    source: snap/local/build-helpers
```

this better matches other parts like curl which we don't want to have to rebuild every time we make a small change elsewhere in the git tree.